inbm/dispatcher-agent/dispatcher/sota/os_updater.py

```diff
@@ -381,3 +392,39 @@ def no_download(self) -> list[str]:
 
     def download_only(self) -> list[str]:
         raise NotImplementedError()
+
+
+class TiberOSUpdater(OsUpdater):
+    """TiberOSUpdater class, child of OsUpdater"""
+    def __init__(self) -> None:
+        super().__init__()
+
+    def update_remote_source(self, uri: Optional[CanonicalUri], signature: Optional[CanonicalUri],
+                             repo: irepo.IRepo) -> List[str]:
+        """Returns empty list for TiberOS --not applicable"""
+        logger.error("SOTA full mode is not supported in TiberOS.")
+        return []
+
+    def update_local_source(self, file_path: str) -> List[str]:
+        """Concrete class method to create command list to update from a local source for TiberOS OS.
+        @param file_path: path to local file
+        @return: Command list to execute to perform update.
+        """
+        return ["echo FIXME: add actual TiberOS command to update from a local source"]
```

Comment:
does this FIXME need to be addressed in this PR?

Reply:
Based on current design, only **download-only** mode and **no-download** mode are supported in TiberOS. No flow for this method, which is used in sota full mode. Will remove the FIXME.

Reply:
Currently all the SOTA requests from INBC are remote source. No support for local source at this moment.
Will removed the FIXME.